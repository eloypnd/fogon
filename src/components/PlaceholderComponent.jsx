import React from 'react'

const PlaceholderComponent = ({data}) => (
  <div style={{textAlign: 'center'}}>
    <h1>Boilerplate</h1>
    <a href='https://en.wikipedia.org/wiki/Single-page_application' target='_blank' rel='noopener noreferrer'>SPA</a><br />
    <small>with</small>
    <p>
      <a href='https://reactjs.org' target='_blank' rel='noopener noreferrer'>create-react-app</a><br />
      +<br />
      <a href='https://redux.js.org' target='_blank' rel='noopener noreferrer'>redux</a>
    </p>
    {data && (
      <small style={{color: '#abc'}}>
        <p>
          &<br /> <a href='https://jsonplaceholder.typicode.com/'>JSONPlaceholder</a>
        </p>
        {data.map(item => <div key={item.id}>{item.name}</div>)}
      </small>
    )}
  </div>
)

export default PlaceholderComponent
