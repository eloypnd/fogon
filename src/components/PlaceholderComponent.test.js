import React from 'react'
import renderer from 'react-test-renderer'
import PlaceholderComponent from './PlaceholderComponent'

it('PlaceholderComponent renders correctly', () => {
  const tree = renderer
    .create(<PlaceholderComponent />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
