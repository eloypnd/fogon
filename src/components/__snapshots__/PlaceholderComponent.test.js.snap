// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceholderComponent renders correctly 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h1>
    Boilerplate
  </h1>
  <a
    href="https://en.wikipedia.org/wiki/Single-page_application"
    rel="noopener noreferrer"
    target="_blank"
  >
    SPA
  </a>
  <br />
  <small>
    with
  </small>
  <p>
    <a
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      create-react-app
    </a>
    <br />
    +
    <br />
    <a
      href="https://redux.js.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      redux
    </a>
  </p>
</div>
`;
